src/app/footer/footer.component.scss

```
@@ -98,9 +97,24 @@
       font: inherit;
       text-decoration: none;
     }
+
+    .social-links {
+      margin-top: 20px;
+      display: flex;
+      flex-direction: row;
```

```suggestion
      flex-direction: row;
      gap: 10px;
```

Spacing between icons.